storage/object/reference-content/ssl-tls-certificate.mdx

```diff
@@ -0,0 +1,174 @@
+content:
+  h1: SSL/TLS Certificates for Object Storage with Edge Services
+  paragraph: Learn how to create and upload an SSL/TLS certificate to serve your bucket content over HTTPS through your customized Edge Services domain, and troubleshoot any potential problems.
+tags: object-storage edge-services ssl-tls certificate ssl tls pem certificate-authority root-certificate pem chain ca
+categories: 
+  - storage
+---
+
+This document contains information to help you create and upload an SSL/TLS certificate to serve your bucket content over HTTPS, through your customized [Edge Services](/storage/object/how-to/get-started-edge-services) domain.
+
+## Introduction
+
+### What is an SSL/TLS certificate? 
+
+An SSL/TLS certificate is a digital certificate that enables an encrypted connection between a client and a web server over HTTPS. 
+
+You may hear certificates referred to as “SSL certificates”, “TLS certificates” or “SSL/TLS certificates”. These are all the same thing. SSL (Secured Socket Layer) was the protocol initially used for encryption, though it has now been replaced with TLS (Transport Layer Security).
+
+SSL/TLS certificates contain a **public key**, which corresponds to a separate **private key**. These work as a pair. When a client wants to establish an encrypted connection to a host, it requests the host's certificate. The host shares the certificate, which includes the public key (the private key is never shared and is kept by the host). The client checks the certificate, and uses the host's public key to encrypt the data that it transfers to the host. The host uses its private key to decrypt the data that has been encrypted by the public key. 
+
+The private key is also used by the host for generating digital signatures, while the public key is used by clients for verifying those signatures.
+
+### When and why do I need an SSL/TLS certificate for Edge Services?
+
+When you enable Edge Services, initially your bucket content is served through the standard Edge Services endpoint, e.g. `https://bucket-name.edge.scw.cloud`. Scaleway's own SSL/TLS certificate, which covers this subdomain, is used to establish the encrypted connection between client and host. If you do not want to customize the standard Edge Services endpoint, you do not need to worry about creating SSL/TLS certificates.
+
+However, if you choose to [customize your Edge Services endpoint with your own subdomain](/storage/object/how-to/get-started-edge-services/#how-to-configure-a-custom-domain), Scaleway's own SSL/TLS certificate can no longer be used to establish encrypted connections. Client connections are now initially going to a different domain which needs to be "guaranteed" by its own certificate (despite the CNAME record for the subdomain pointing to the Scaleway endpoint). 
```

Comment:
To me, even if you customize the edge services endpoint, technically you can still use the default Scaleway endpoint in svc.edge.scw.cloud.

Reply:
I see what you mean, I've edited to avoid this potential misunderstanding